language-design.md

```diff
@@ -102,6 +201,18 @@ you do so? If not, why not?
 
 **Response**
 
+We think that this shows that there is a balance to be struck between natural
+language design and regular scripting language syntax.  It is also important
+to know the audience of the language.  If they are experienced programmers it
+is probably easier for them if your programming language looks more like what
+they are used to and has the less natural syntax of Java or C, but if they are
+less experienced it would feel more natural to the user to use a more natural
+language type of syntax and grammer.
```

Comment:
The issue we see for something to feel more natural (language-wise) is that when you run into a bug, you would use your natural language instincts to try to solve it (and thus run into more issues) rather than using your programming language instincts. For novices it might be better to separate the natural language context from the programming language context so that they are not confused, i.e. don't learn badminton and tennis at the same time as first uses the wrist and if you use your wrist in tennis you won't have enough power and might hurt yourself.